client/src/app/plants/plant.component.html

```diff
@@ -11,4 +11,13 @@ <h3 >{{plant.cultivar}}</h3>
     <div *ngIf="rated">
         Thanks for your feedback!
     </div>
+    <br>
+    <div *ngIf="!commented">
+        <textarea [(ngModel)]="currentQuery" id="commentBox" rows="5" wrap="on" style="width: 95%;"></textarea>
```

Comment:
Thought: do we want to have a count in the Plants metadata of how many comments each flower gets? We'd just need to add another field to the metadata in the plant database, and then increment each time a comment is left, which wouldn't be too hard.

Or do we not care at this point, since we can look at the comments individually anyway?